lib/platform/github/index.ts

```diff
@@ -1662,6 +1667,7 @@ export async function createPr({
       url: 'https://github.com/renovatebot/renovate',
     });
   }
+  pr.draft = draftPR;
```

Comment:
```suggestion
```

This is unnecessary and unused?

Reply:
I am using it in tests and could be used for later. Do you still want me to remove it?

Reply:
Removed this in latest commit fe70f80